.github/workflows/testing_minimum.yml

```diff
@@ -67,6 +67,8 @@ jobs:
                 python -m pip install --upgrade pip setuptools
                 pip install ${{ matrix.min_dep }}
         -   name: Install nilearn
-            run: pip install .[plotting,plotly,test]
+            run: |
+                pip install .[plotting,plotly,test]
+                pip install pytest-mpl
```

Comment:
should fix some tests failures on main: see https://github.com/nilearn/nilearn/actions/runs/13450777115